dspace-api/src/main/java/org/dspace/authority/AuthorityValue.java

```diff
@@ -468,16 +468,14 @@ public void updateConceptFromAuthorityValue(Concept concept) throws SQLException
                 for(String value : otherMetadata.get(name))
                 {
                     // Add it to the list
-                    concept.addMetadata(schema,element,qualifier,"",value,null,-1);
+                    concept.addMetadata(context, schema,element,qualifier,"",value,null,-1);
```

Comment:
The -1 for confidence seems like it should either be a constant or the default value in the schema